scripts/add-nix-gcroots.sh

```diff
@@ -12,6 +12,6 @@ GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
 rm -rf .nix-gcroots
 mkdir .nix-gcroots
 
-drv=$(nix-instantiate --argstr target-os all --add-root ${GIT_ROOT}/shell.nix)
+drv=$(nix-instantiate --add-root ${GIT_ROOT}/shell.nix)
```

Comment:
But I did...

Reply:
No sure I understand your answer.

Reply:
But `target-os`(or now `target`) is irrelevant unless used with `/shell.nix`.

Reply:
And we are instantiating `${GIT_ROOT}/shell.nix`, so it would make sense to still pass `target all` to ensure we're pinning as many derivations as possible, so they don't get deleted by a garbage collection. If we pin `target none`, there'll be very few derivations pinned.

Reply:
Okay, fair enough, fixed.